storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3007,6 +3007,47 @@ class Rdb_snapshot_status : public Rdb_tx_list_walker {
           tx->get_delete_count());
     }
   }
+
+  void populateDeadlockBuffer() {
+    uint max_key_size = 50;
+    m_data += "---------\n"
+              "DEADLOCKS\n"
+              "---------\n"
+              "LIST OF DEADLOCKS FOR EACH SESSION:\n";
```

Comment:
If i'm not mistaken the 'list of snapshots for each session' actually shows which snapshots are from which session via the mysql thread id. 

However in your case there isn't such a separation.  It looks like the number next to DEADLOCK is an incremented counter.  So I think maybe we just say "LIST OF DEADLOCKS".